### PR TITLE
Fix RAAL timer drift

### PR DESCRIPTION
**Pull request by hubertmis**

### CODE REVIEW

**Comment by hubertmis** [comment]
Regression test

File: src/rsch/raal/softdevice/nrf_raal_softdevice.c [hunk 1]
```diff
@@ -145,9 +151,29 @@ static timer_action_t m_timer_action;
...
 /**@brief Number of already performed extentions tries on failed event. */
 static volatile uint16_t m_timeslot_extend_tries;
 
+/***************************************************************************************************
+ * @section Drift calculations
+ **************************************************************************************************/
+
+static uint32_t time_corrected_for_drift_get(uint32_t time)
+{
+    return time - NRF_802154_DIVIDE_AND_CEIL(time * m_config.lf_clk_accuracy_ppm, PPM_UNIT);
```
[the remark this answers is not in the document]
Comment (hubertmis) [reply src/rsch/raal/softdevice/nrf_raal_softdevice.c]: I like this idea - I've forgotten our discussion.
If I remember correctly the same accuracy (40 PPM) is needed for 802.15.4.

File: src/rsch/raal/softdevice/nrf_raal_softdevice.c [hunk 2]
```diff
@@ -88,8 +91,8 @@ _Pragma("GCC diagnostic pop")
 /**@brief RAAL Timer interrupt number. */
 #define RAAL_TIMER_IRQn                              TIMER0_IRQn
 
-/**@brief Maximum jitter relative to the start time of START and TIMER0 (safety margin) events. */
-#define TIMER_TO_SIGNAL_JITTER_US                    NRF_RADIO_START_JITTER_US + 6
+/**@brief Minimum time prior safe margin reached by RTC when TIMER reports reached margin. */
```
Comment (KaSroka) [inline src/rsch/raal/softdevice/nrf_raal_softdevice.c]: Add unit in comment

**Comment by hubertmis** [comment]
Regression test

**Pull request merged**